src/common/chat/autocompleter.test.ts

```diff
@@ -66,6 +66,17 @@ test('Autocompleter returns all subpaths', () => {
   expect(results).toContain('diofarms');
 });
 
+test('Autocompleter will find full token if no prefix matches', () => {
```

Comment:
this is more of a question of behavior, but depending on what you think makes more sense it may make sense to add a test verifying that asking for 'cabbages' on

['diocletian', 'farms', 'cabbages', 'beautiful cabbages'];

uses the prefix match first and returns just ['cabbages'] rather than ['cabbages', 'beautiful cabbages']

Reply:
100%, added. And to me I think returning just cabbages is the right behavior in that case.